blogs/dns_tracking/dns_tracking.md

```diff
@@ -61,13 +59,67 @@ collector object with the following fields enabled in eBPF config
 section
 
 ```yaml
+apiVersion: flows.netobserv.io/v1beta1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
   agent:
     type: EBPF
     ebpf:
       privileged: true
-      enableDNSTracking: true
+      features:
+        - DnsTacking
 ```
 
+## A quick tour in the UI
+
+Once `DnsTacking` feature enabled, the Console plugin will automatically adapt to provide
+additionnal filters and show informations across views.
+
+Open your OCP Console and move to 
+`Administrator view` -> `Observe` -> `Network Traffic` page as usual.
+
+Three new filters, `DNS Id`, `DNS Latency` and `DNS Response Code` will be available 
+in the common section:
+
+![dns filters](./images/dns_filters.png)
+
+The first one will allow you to filter on a specific DNS Id to correlate with your query.
+
+![dns id](./images/dns_id.png)
+
+The second one helps to identify potential performance issues by looking at DNS resolution latency.
+
+![dns latency more than](./images/dns_latency_more_than.png)
+
+The third filter surfaces DNS response codes, which can help detect errors or unauthorized resolutions.
```

Comment:
what does "unauthorized resolutions" mean? or should we simplify say "which can help detect DNS error sources"

Reply:
Unauthorized it’s one of the response code in the flags